.github/pull_request_template.md

```diff
@@ -14,16 +14,18 @@ Please check all options that are relevant
 - [ ] Documentation update only  
 
 # How Has This Been Tested?  
-
 **Tested Configuration**:  
-- [ ] CentOS, Specify version(s):  
-- [ ] PostgreQL, Specify version(s):  
-- [ ] docs tested with hugo <0.60  
+- [ ] Ansible, Specify version(s):  
+- [ ] Specify Distro & version(s):  
+- [ ] Specify Package distro & version(s):  
+- [ ] Packages compiled from source & their version:  
+- [ ] PostgreSQL, Specify version(s):  
+- [ ] docs tested with hugo version(s):  
 
 Tested with playbook(s):  
```

Comment:
```suggestion
Tested with playbook(s):  

```

This tends to be crowded up with the next section and more than once I missed this section early on.